percy/providers/appAutomateProvider.js

```diff
@@ -95,17 +95,26 @@ class AppAutomateProvider extends GenericProvider {
 
   // Override this for AA specific optimizations
   async getTiles(fullscreen, fullPage, screenLengths, scrollableXpath, scrollableId) {
-    // Temporarily restrict AA optimizations only for full page
-    if (fullPage !== true) {
+    // Override AA optimizations
+    if (this.isDisableRemoteUpload()) {
       return await super.getTiles(fullscreen, fullPage, screenLengths);
     }
 
+    let screenshotType = 'fullpage';
+    let projectId = 'percy-prod';
+    if (fullPage !== true) {
+      screenshotType = 'singlepage';
+    }
+    if (this.isPercyDev()) {
+      projectId = 'percy-dev';
+    }
```

Comment:
Test for this is missing

Reply:
I had added tests within `getTiles` describe. I added separate describe for the tests.